portfolio/src/main/webapp/style.css

```
@@ -1,9 +1,85 @@
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin: 20px;
 }
+#nav_bar{
```

[4.2.7](https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation): Separate rules by new lines.

```suggestion
}

#nav_bar {
```

Here and elsewhere.